cuda/matrix/coo_kernels.cu

```diff
@@ -49,8 +49,23 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
 namespace gko {
+/**
+ * @brief The device specific kernels.
+ *
+ * \ingroup kernels
```

Comment:
should that be "@ingroup" ?
same for all following ones...

Reply:
I think `\ingroup` and `@ingroup`  are equivalent. The doxygen manual says [`\ingroup`](http://www.doxygen.nl/manual/grouping.html)

Reply:
They should be equivalent, but I think it does not hurt to have it uniform.

Reply:
Yes, I will do that.

Reply:
You marked this as resolved though this commit does the opposite (uniformization towards `\`). https://github.com/ginkgo-project/ginkgo/pull/258/commits/9f8f30630800812940aef2dde6d82ecb5a48ffa5